octoprint_mrbeam/util/material_csv_parser.py

```diff
@@ -3,6 +3,7 @@
 
 MRBEAM = "Mr Beam II"
 MRB_DREAMCUT = "MrB II Dreamcut"
+MRB_DREAMCUT_S = "MrB II Dreamcut S"
 MRB_READY = "MrB II Dreamcut Ready"  # not used yet
 MRB_DREAMCUT_NOT_VALIDATED = "Dreamcut (not validated)"
```

Comment:
Actually you might need to add `"Dreamcut S (not validated)"` to the invalid keys... but perhaps it already filters out all of the strings containing `"not validated"` ... Do you know?

Reply:
It is already filtered out but for the matter of consistency and to remember it exists in the csv file I'll add it here